terraform/test-fixtures/apply-destroy-nested-module-with-attrs/middle/middle.tf

```diff
@@ -0,0 +1,8 @@
+variable param {}
```

Comment:
technically valid to leave the var name bare, but conventionally we always quote it -> `variable "param" {}`